docs/_Docs_ReputationMining.md

```diff
@@ -4,4 +4,55 @@ section: Docs
 order: 9
 ---
 
-Pending finalization of the reputation system implementation (see [GitHub](https://github.com/JoinColony/colonyNetwork) for more information).
+This is an overview of the process by which Reputation scores in Colony are maintained.
+
+For a complete description of the Reputation Mining process, see the [whitepaper](https://colony.io/whitepaper.pdf)
+
+# Rationale for off-chain Reputation
+Events in Colony that affect reputation are numerous and expected to be frequent. The calculations for global reputation updates would exceed Ethereum's gas limit, and cannot be performed on-chain. However, Reputation doesn't *need* to be on-chain, because reputation is entirely deterministic and calculated from on-chain events.
+
+So, rather than calculating reputation scores on-chain, users submit to the colony contract a transaction  which contains their score, together with a proof that the score is consistent with the global state of reputation.
+
+## Reputation Mining Overview
+The process of maintaining consensus over the global state of reputation is called *Reputation Mining*.
+
+Reputation Mining resembles a proof-of-stake blockchain consensus protocol. Miners must put up a stake of the network token, CLNY, to participate in the reputation mining process, whereby the global state of reputation is calculated off-chain and submitted to the blockchain as a `reputationRootHash`. In the case of honest submissions, the miner is rewarded reputation in the Meta Colony tagged with a special 'mining' skill. In the event of malicious activity, a miner with a false submission will be punished by having her stake slashed.
+
+The process works on an “innocent until proven guilty” principle: Although it’s impossible for a valid hash to be proven true on-chain, it’s relatively straightforward for an invalid hash to be proven false. In the event that two submitted hashes differ, the false one can be fleshed out through an automated challenge process *on-chain* that will eventually zero in on the exact on-chain events that differ between the two submitted hashes.
```

Comment:
Not impossible, just very expensive. 

I'd also make it clear somewhere in this page that the narrative it supplies is assuming two things:

* Exactly two reputation hashes are submitted
* Exactly one of those hashes is wrong, and the other is correct.

In reality, an unlimited number of hashes can be submitted, and none of them are guaranteed to be right.
We are assuming that the correct hash is one of the (potentially many) submitted. It is possible that two incorrect hashes will be played off against each other, and one will 'win' that challenge. That doesn't mean that it's correct hash, just that it made a mistake in its reputation calculation later than the losing hash.